Cakefile

```diff
@@ -40,9 +40,38 @@ visitDirectory = (directory, visitor) ->
     visitor(filepath)
 
 task "build", "compile all coffeescript files to javascript", ->
+  fs.unlink "pages/git-information.js", (->)
   coffee = spawn "coffee", ["-c", __dirname]
   coffee.on 'exit', (returnCode) -> process.exit returnCode
 
+appendToOptionsPage = (data) -> fs.writeFile "pages/git-information.js", data
+
+task "dev-build", "compile all coffeescript files and add git information", ->
+  invoke "build"
+
+  branchDone = commitDone = false
```

Comment:
This could use a one-line comment explaining what's happening: "we want to show the user what dev version they're running via the options page" and reference #1350.